### PR TITLE
Add Navbar to the website

### PR DESCRIPTION
**Pull request by Shubh4m-B**
## Please, go through these steps before you submit a PR.

- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

### CODE REVIEW

**Comment by Shubh4m-B** [comment]
We have tried to solve issue #4. The navbar comes into view after the page is scrolled by a certain height.


Navbar

**Comment by Shubh4m-B** [comment]
Also added a hamburger icon for toggling the sidebar.


Navbar minimized

File: Gemfile.lock [hunk 1]
```diff
@@ -53,13 +55,22 @@ GEM
     safe_yaml (1.0.5)
     sassc (2.4.0)
       ffi (~> 1.9)
+    sassc (2.4.0-x64-mingw32)
+      ffi (~> 1.9)
     terminal-table (1.8.0)
       unicode-display_width (~> 1.1, >= 1.1.1)
+    thread_safe (0.3.6)
```
Comment (grace-omotoso) [inline Gemfile.lock:62]: Please, remove Gemlock file changes

File: _sass/navbar.scss [hunk 2]
```diff
@@ -1,36 +1,177 @@
...
+  justify-content: center;
+  align-items: center;
 }
 
 .nav-content .logo-touch-target {
-    display: inline-flex;
-    height: 60%;
-    width: 350px;
+  display: inline-flex;
+  //   height: 60%;
```
Comment (grace-omotoso) [inline _sass/navbar.scss:21]: remove commented code

File: _sass/navbar.scss [hunk 3]
```diff
@@ -1,36 +1,177 @@
...
+  width: 350px;
 }
 
 .nav-content .nav-logo {
-    height: 60px;
-    margin: auto;
+  height: 60px;
+  margin: auto;
```
Comment (grace-omotoso) [inline _sass/navbar.scss:27]: Is this responsive? Did you test on various devices?

File: _sass/navbar.scss [hunk 4]
```diff
@@ -1,36 +1,177 @@
...
-    text-align: right;
-    padding-top: 0px;
-    padding-right: 0px;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    list-style: none;
+    // background-color: aquamarine;
+    li {
+      // background-color: #74b4fe;
```
Comment (grace-omotoso) [inline _sass/navbar.scss:75]: Remove commented code

File: _sass/navbar.scss [hunk 5]
```diff
@@ -1,36 +1,177 @@
...
+  // background-color: #74b4fe;
+  height: 100%;
+  width: 90%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+nav {
+  // background-color: #133667;
```
Comment (grace-omotoso) [inline _sass/navbar.scss:62]: remove commented code

**Review by grace-omotoso — CHANGES_REQUESTED** [review]
Please, see my comments

**Comment by grace-omotoso** [comment]
The user experience with this implementation needs to be improved. Users should not have to scroll to a certain height before they see the navbar. I am not able to see the navbar because I use a big screen. The navbar should be displayed immediately after the page is opened.

**Comment by akshitadixit** [comment]
> The user experience with this implementation needs to be improved. Users should not have to scroll to a certain height before they see the navbar. I am not able to see the navbar because I use a big screen. The navbar should be displayed immediately after the page is opened.

Hi @grace-omotoso we only added the view on scroll only because we found it in the issue #4 description:
>As the portfolio gains more information, it may make sense to add a navigation bar. This bar should include the logo on the left side but only once the page has scrolled down a bit.

**Pull request closed without merging**